eng/Versions.props

```diff
@@ -114,7 +114,7 @@
     <NewtonsoftJsonVersion>12.0.3</NewtonsoftJsonVersion>
     <MoqVersion>4.12.0</MoqVersion>
     <!-- Docs -->
-    <MicrosoftPrivateIntellisenseVersion>3.1.0-preview-20200129.1</MicrosoftPrivateIntellisenseVersion>
+    <MicrosoftPrivateIntellisenseVersion>3.0.0-preview-20200602.3</MicrosoftPrivateIntellisenseVersion>
```

Comment:
Why are we downgrading the minor version?

Reply:
We aren't. It seems the Docs build generates the package with that string. We can ask them to change it so it doesn't cause confusion. Santi and I checked the contents and the `net` folder contains the latest APIs, even considering the package has that name.

Reply:
Because we before used to produce this packages from corefxlab, now they're produced by the docs team to a different feed and this is the version they're using.

Reply:
would it make sense to ask them to align their version with our product version?

Reply:
Yes, I'll ask them to change it because it's confusing. But for now we can use this.